buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -48,16 +48,17 @@ source activate $CONDA_ENV
 set -v
 
 # gitpython needed for CI testing
-if [ $PYTHON \< "3.8" ]; then
-    $CONDA_INSTALL gitpython
-else
-    $PIP_INSTALL gitpython
-fi
+$CONDA_INSTALL gitpython
 
 # Install optional packages into activated env
 if [ "${VANILLA_INSTALL}" != "yes" ]; then
     # Scipy, CFFI, jinja2, IPython and pygments are optional dependencies, but exercised in the test suite
-    $CONDA_INSTALL ${EXTRA_CHANNELS} cffi scipy jinja2 ipython pygments
+    $CONDA_INSTALL ${EXTRA_CHANNELS} cffi jinja2 ipython pygments
+    if [[ "$PYTHON" == "3.8" &&  $(uname) == Darwin ]]; then
+        $PIP_INSTALL scipy
```

Comment:
Oh, in gh-5029 I didn't notice `defaults` hadn't builds on all platforms yet.
FYI: They added those a couple of days ago.